gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
```

Comment:
Java 21 cannot run Gradle 8.4: via https://docs.gradle.org/current/userguide/compatibility.html#java

Makes it difficult to debug in IntelliJ.